mock-identity-system/src/main/resources/application-local.properties

```diff
@@ -1,6 +1,12 @@
+##-----------------------------------------Mock-identity-system properties----------------------------------------------
+
+mosip.mock.ida.identity.schema.url=classpath:/mock-identity-schema.json
+mosip.mock.ida.update-identity.non-mandatory.fields={"fullName","name","givenName","familyName","middleName","nickName","preferredUsername","gender","streetAddress","locality","region","country","pin","preferredLang","dateOfBirth","postalCode","encodedPhoto","email","phone","zoneInfo","locale","password"}
+
```

Comment:
If the property is defined in the application-default.properties file, and no changes are required in the local profile, then there is no need to redefine the same property in application-local.properties.

Reply:
removed